public/locales/ja-JP/translations.json

```diff
@@ -6,7 +6,7 @@
   "assets.no_issued_message": "トークンが見つかりません。",
   "assets.no_nfts_message": "NFTが見つかりません。",
   "network": "ネットワーク",
-  "amendments": null,
+  "amendments": "Amendment",
```

Comment:
Why are some words written out in English?

Reply:
Some words are commonly used in English in the Japanese community.

In particular, “Amendment” is an English word that users from non-English speaking countries often see (through services such as XRPScan).